packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/EnumTest.m

```diff
@@ -15,14 +15,14 @@ @interface EnumTest : XCTestCase
 @implementation EnumTest
 
 - (void)testEcho {
-  ACDataWithEnum *data = [[ACDataWithEnum alloc] init];
-  data.state = ACEnumStateError;
+  DataWithEnum *data = [[DataWithEnum alloc] init];
```

Comment:
How come we lost all the prefixes to class names?

Reply:
From the PR description:

> ObjC prefixes are no longer used, since collisions were eliminated from the input pigeons during the Swift generator bringup.